Classes/Service/SolrServiceProvider.php

```diff
@@ -627,6 +627,7 @@ protected function createQuery(): void
         $this->query = $this->connection->createSelect();
         $this->addFeatures();
         $this->addTypoScriptFilters();
+        $this->addMainQueryOperator();
```

Comment:
The naming of "MainQuery" is quite misleading here - I guess "DefaultQuery" is what expresses the method best